packages/dbml-core/src/parse/Parser.js

```diff
@@ -7,6 +8,10 @@ import mssqlParser from './mssqlParser';
 import { parse } from './ANTLR/ASTGeneration';
 
 class Parser {
+  constructor (DBMLCompiler = undefined) {
```

Comment:
If a param is not passed when called, it will already be `undefined`
  constructor (DBMLCompiler) {
```